src/Sulu/Bundle/MediaBundle/Resources/js/containers/MultiMediaDropzone/MultiMediaDropzone.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+import React from 'react';
+import type {ElementRef} from 'react';
+import {observer} from 'mobx-react';
+import {action, observable} from 'mobx';
+import Dropzone from 'react-dropzone';
+import MediaUploadStore from '../../stores/MediaUploadStore';
+import MediaItem from './MediaItem';
+import DropzoneOverlay from './DropzoneOverlay';
+
+type Props = {
+    children: any,
+    locale: observable,
```

Comment:
This is making problems 😉  As already said, that only happens in newer versions of mobx, you probably have an old one installed.